gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -67,13 +66,15 @@ public void apply(Project project) {
             String version = Optional.ofNullable(getClass().getPackage().getImplementationVersion())
                     .orElse("latest.release");
 
-            Configuration refasterConfiguration = project.getConfigurations().create(REFASTER_CONFIGURATION);
+            Configuration refasterConfiguration = project.getConfigurations().create("refaster", conf -> {
```

Comment:
Could we add a quick test for this? All we have to do is add any dependency to the refaster configuration and validate that compile was a no-op

Reply:
Annoyingly I can't just add any-old dependency (we validate too hard).  Needs to be a real refaster source dep.